tests/beaker_tests/lib/utilitylib.rb

```diff
@@ -357,6 +323,19 @@ def system_default_switchport(agent, state=false,
   end
 end
 
+# Helper for checking/setting 'system default switchport'
+def config_system_default_switchport?(tests, id)
```

Comment:
This is find, but did you consider combining `config_system_default_switchport?` and `config_system_default_switchport_shutdown?` since the code is identical but add a `key` argument?

`def config_system_defaults?(tests, id, key)`